test/test_basics.cpp

```diff
@@ -75,9 +76,10 @@ namespace goldilock::test {
 
     // note: on windows we could test up to full 16bits length, but this is kinda overkill already
     for(size_t ret = 0; ret < 255; ret++) {
-      auto result = run_goldilock_command_in(wd, "--lockfile", "test.lock", "--", "exit", std::to_string(ret));
+      auto result = run_goldilock_command_in(wd, "--lockfile", "test.lock", "--" , "exit", std::to_string(ret));
+      std::cout<<"retrun code is "<<result.return_code<<" and expected is " <<ret<<std::endl;
```

Comment:
I'd rather have this checked with https://www.boost.org/doc/libs/1_87_0/libs/test/doc/html/boost_test/utf_reference/testing_tool_ref/assertion_boost_level_eq.html to avoid typos and have a single check with a message containing the inputs.